lib/github_core.ml

```diff
@@ -213,6 +213,37 @@ module Make(CL : Cohttp_lwt.Client) = struct
     let repo_deploy_key ~user ~repo ~num =
       Uri.of_string (Printf.sprintf "%s/repos/%s/%s/keys/%d" api user repo num)
```

Comment:
Would not be meaningful to have a `uri_format` function that we could use to rewrite

    Uri.of_string (Printf.sprintf "%s/repos/%s/%s/keys/%d" api user repo num)

as

    format_uri "%s/repos/%s/%s/keys/%d" api user repo num

This would have ten uses below!